tests/lib/rules/no-multiple-empty-lines.js

```diff
@@ -61,8 +65,16 @@ ruleTester.run("no-multiple-empty-lines", rule, {
             code: "`\n\n`",
             options: [{ max: 0 }],
             ecmaFeatures: { templateStrings: true }
-        }
+        },
 
+        {
+            code: "// valid 5\nvar a = 5;\n\n\n\n",
+            options: [ { max: 0, maxEOF: 4 } ]
+        },
+        {
+            code: "// valid 5\nvar a = 5;\n\n\n\n",
```

Comment:
When `maxEOF` is not specified, the current behavior is not changed (this means the last `\n` is not counted, so `\n\n\n\n` only counts as 3). From comments in the code, I believe this behavior has its roots in the way some editors do not show the last `\n`.